pkg/storage/storage_handler.go

```diff
@@ -0,0 +1,44 @@
+package storage
+
+import (
+	"github.com/cyberark/conjur-authn-k8s-client/pkg/storage/config"
+)
+
+type SecretsHandler interface {
+	HandleSecrets() error
+}
+
+type AccessTokenHandler interface {
+	Read()	([]byte, error)
+	Write()	error
+	Delete()	error
+}
+
+// Need underscore here unsure why
+type Storage_Handler struct {
+	AccessToken AccessTokenHandler
```

Comment:
Defined the members as pointers like you did in all other structs

Reply:
After discussion with Moti we realize that interfaces do not need to be specified with a * because they are implicit